x-pack/plugins/cases/public/components/severity/config.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useEuiTheme, EuiHealth } from '@elastic/eui';
+import { CaseSeverity } from '../../../common/types/domain';
+import { CRITICAL, HIGH, LOW, MEDIUM } from './translations';
+
+interface Props {
+  severity: CaseSeverity;
+}
+
+export const severities = {
+  [CaseSeverity.LOW]: {
+    label: LOW,
+  },
+  [CaseSeverity.MEDIUM]: {
+    label: MEDIUM,
+  },
+  [CaseSeverity.HIGH]: {
+    label: HIGH,
+  },
+  [CaseSeverity.CRITICAL]: {
+    label: CRITICAL,
+  },
+};
+
+export const SeverityHealth: React.FC<Props> = ({ severity }) => {
+  const { euiTheme } = useEuiTheme();
+
+  const severityData = {
+    low: { color: euiTheme.colors.vis.euiColorVis0, label: LOW },
```

Comment:
The vis colors have changed and don't directly map to similar colors ([related conversation](https://github.com/elastic/kibana/pull/203518#discussion_r1883000624))

![image](https://github.com/user-attachments/assets/ef94bc3e-292e-491a-ad67-8767d74408e1)

Since these are used as severity colors here, you likely want to ensure they are still using a specific color.
The severity color palette is still [being worked on](https://github.com/elastic/kibana/issues/203387), but we can remap vis colors per theme.
I'm not sure who's the responsible designer to involve here for this code? cc @gvnmagni 

We have two options here:
- use new tokens which mean the current Amsterdam colors change a bit (the main issue seems to be distinguishing high and critical)
- map severity colors per theme

**Option 1**

```ts
low: { color: euiTheme.colors.vis.euiColorVisSuccess0, label: LOW }
medium: { euiTheme.colors.vis.euiColorVisWarning0, label: MEDIUM }
high: { color: euiTheme.colors.vis.euiColorVisDanger1, label: CRITICAL }
critical: { color: euiTheme.colors.vis.euiColorVisDanger0, label: CRITICAL }
```

| Severity level | previous value | Amsterdam | Borealis |
|---------------|------------|----------|----------|
| low | `#54B399` | `#209280` | `#24C292` |
| medium | `#D6BF57` | `#D6BF57` | `#FCD883` |
| high | `#DA8B45` | `#E7664C` | `#FFC9C2` |
| critical | `#E7664C` | `#CC5642` | `#F6726A` |

**Option 2**

```ts
const { euiTheme } = useEuiTheme();
const { hasVisColorAdjustment } = euiTheme;

low: { color: hasVisColorAdjustment ? euiTheme.colors.vis.euiColorVis0 : euiTheme.colors.vis.euiColorVisSuccess0, label: LOW }
medium: { color: hasVisColorAdjustment ? euiTheme.colors.vis.euiColorVis5 : euiTheme.colors.vis.euiColorVisWarning0, label: MEDIUM }
high: { color: hasVisColorAdjustment ? euiTheme.colors.vis.euiColorVis7 : euiTheme.colors.vis.euiColorVisDanger1, label: CRITICAL }
critical: { color: hasVisColorAdjustment ? euiTheme.colors.vis.euiColorVis9 : euiTheme.colors.vis.euiColorVisDanger0, label: CRITICAL }
```
| Severity level | previous value | Amsterdam | Borealis |
|---------------|------------|----------|----------|
| low | `#54B399` | `#54B399` | `#24C292` |
| medium | `#D6BF57` | `#D6BF57` | `#FCD883` |
| high | `#DA8B45` | `#DA8B45` | `#FFC9C2` |
| critical | `#E7664C` | `#E7664C` | `#F6726A` |

Reply:
Hi @mgadewoll, thanks for the detailed suggestion here. @joana-cps is the designer in charge of this area of the product. I'm personally ok with either option. Will defer to @gvnmagni or @joana-cps.

Reply:
@joana-cps let's chat about this, I want to be sure I can support you but I need to understand a little bit better the context

Reply:
Hi @mgadewoll, thanks very much for the detailed suggestion here, this was a topic that we spent some time discussing with the team (@georgianaonoleata1904 and @cnasikas) and we agreed on using this approach:

Severity level | previous value | Amsterdam | Borealis | Color token
-- | -- | -- | -- | --
low | `#54B399` | `#00BFB3` | `#008A5E` | colors.success
medium | `#D6BF57` | `#FEC514` | `#FACB3D` | colors.warning
high | `#DA8B45` | `#F04E98` | `#BC1E70` | colors.accent
critical | `#E7664C` | `#BD271E` | `#C61E25` | colors.danger

The main reasons:
- We tried both proposals but in Borealis `#FFC9C2` it's too light for a High Status
- We use `colors.success`, `colors.warning` and `colors.danger` in other ResponseOps pages, so we would like to have some consistency in our status
- We use same color token for both themes
- The new severity colors change will have less impact since we are already using dark red/pink in higher status

We were missing an "orange" for the high status, and this was the best solution that we could find
Let me know what you think 😊

Reply:
ok, I got the chance to chat with Joana and I believe we reached a consensus. 

From my perspective, I would not change anything in Amsterdam, while for Borealis I would proceed this way. We don't have yet all the tokens set up for our new Severity Color Palette ([issue](https://github.com/elastic/kibana/issues/203387)) but what we can do is to apply color primitives directly and eventually come back here introducing severity tokens as soon as they are ready.

Specifically, this would be the mapping of our levels in this case:

<img width="300" alt="Screenshot 2024-12-17 at 13 00 30" src="https://github.com/user-attachments/assets/c93df1fa-ea42-4e31-87ca-1df3bad5ca35" />

Reply:
@gvnmagni Color primitives are not available on the theme, they are baseline definitions, not tokens.
That means we'd need to use static colors if we'd wanted to do this 🙈

Reply:
Thank you Lene, you are right. I would suggest though to hard-code this values so that there will be no change for the user in the end in Borealis. We'll just need to get back here once we have all Severity tokens in place and refer to them

<img width="400" alt="Screenshot 2024-12-17 at 15 11 34" src="https://github.com/user-attachments/assets/c6fb6486-82b8-4c6e-a972-626c5816e94c" />

Reply:
These colors, even if nor final, are already being considered in other places (pros and cons of Elastic open way of working) and I would prefer to head in this direction, even if not super final, instead of changing things many times. This will get us closer to consistency across solutions

Reply:
@joana-cps Since we will use the hardcoded values, should we use the same in Borealis? The colors on your table seem to be different from the two themes.

Reply:
Ok, is this a final decision? Could we all agree on this before @georgianaonoleata1904 does the implementation?

Reply:
@mgadewoll can you please confirm if you're ok with these changes?

Reply:
Yes, I agree. Let's keep Amsterdam as is and use the new ones for Borealis.

Fyi, we're introducing [new tokens for severity colors](https://github.com/elastic/eui/pull/8247). These will then replace these mappings, but it will be done as a follow-up.